Extend/Controller/OrderController.php

```diff
@@ -175,7 +175,17 @@ public function execute()
         $oOrder = oxNew(Order::class);
         $sOrderId = Helper::getSessionChallenge();
         $bIsOrderLoaded = $oOrder->load($sOrderId);
-
+        $sWdPaymentRedirect = Registry::getRequest()->getRequestParameter('wdfinishedpayment');
```

Comment:
Encapsulate the loading of the order into a method.

Reply:
👍